administrator/components/com_menus/models/item.php

```diff
@@ -1273,7 +1273,8 @@ public function save($data)
 		$dispatcher = JEventDispatcher::getInstance();
 		$pk         = (!empty($data['id'])) ? $data['id'] : (int) $this->getState('item.id');
 		$isNew      = true;
-		$table      = $this->getTable();
+		JModelLegacy::addTablePath(JPATH_ADMINISTRATOR . '/components/com_menus/tables');
```

Comment:
Hint: You will not needs to change anything in this model file. except `require_once JPATH_ADMINISTRATOR . '/components/com_menus/helpers/menus.php';`